doc/source/cheatsheet/cheatsheet.qmd

```diff
@@ -169,13 +170,13 @@ shutil.copyfile(out_file_src, out_file_des)
 This client script retrieves all .out files from the server's working directory:
 
 ```{python}
-wb.download_file("*.out")
+workbench.download_file("*.out")
 ```
 
 Use the ``{download_project_archive()}`` function to save, archive, and download the current Workbench project from the server to the client:
```

Comment:
Should the brackets be in the function tagged entity? I've never seen this before.